spec/requests/api/v1/events/events_spec.rb

```diff
@@ -82,7 +76,27 @@
       expect(response).to match_response_schema(:event)
     end
 
-    it 'returns and error message when invalid' do
+    it 'returns an error when the event cant be found' do
+      invalid_id = 666
```

Comment:
I'd pass up something like 'foo' instead of a real number because as you run tests, your IDs can increment to the point where this is actually a passing test.